src/v2/ion/RemediationConstants.js

```diff
@@ -42,6 +42,68 @@ const FORMS = {
   TERMINAL: 'terminal',
 };
 
+const FORMNAME_CLASSNAME_MAPPING = {
+  'identify': ['identify', 'primary-auth'],
+  'identify-password': ['identify', 'primary-auth'],
+  'select-identify': ['select-identify'],
+  'enroll-profile': ['enroll-profile', 'registration'],
+
+  // 1st step of the challenge flow for sms/call
+  'authenticator-verification-data-sms': ['authenticator-verification-data-sms'],
+  'authenticator-verification-data-phone': ['authenticator-verification-data-phone'],
+
+  'challenge-authenticator-voice': ['challenge-authenticator-voice', 'mfa-verify-passcode'],
+  'challenge-authenticator-sms': ['challenge-authenticator-sms', 'mfa-verify-passcode'],
+  'challenge-authenticator-email': ['challenge-authenticator-email', 'mfa-verify-passcode'],
+  'challenge-authenticator-password': ['challenge-authenticator-password', 'mfa-verify-password'],
+  'challenge-authenticator-security_question': ['challenge-authenticator-security_question', 'mfa-verify-question'],
+
+  'enroll-authenticator-voice': ['enroll-authenticator-voice', 'enroll-call'],
+  'enroll-authenticator-sms': ['enroll-authenticator-sms', 'enroll-sms'],
+  'enroll-authenticator-email': ['enroll-authenticator-email', 'enroll-email'],
+  'enroll-authenticator-password': ['enroll-authenticator-password', 'enroll-password'],
+  'enroll-authenticator-security_question': ['enroll-authenticator-security_question', 'enroll-question'],
+
+  'select-authenticator-authenticate': ['select-authenticator-authenticate'],
+  'select-authenticator-enroll': ['select-authenticator-enroll', 'enroll-choices'],
+
+  'select-authenticator-enroll-data': ['select-authenticator-enroll-data'],
+  'authenticator-enrollment-data': ['authenticator-enrollment-data'],
+
+  'reenroll-authenticator': ['reenroll-authenticator', 'password-expired'],
+  'reenroll-authenticator-warning': ['reenroll-authenticator-warning'],
+  'reset-authenticator': ['reset-authenticator'],
+  'skip': ['skip'],
+
+  'success-redirect': ['success-redirect'],
+  'redirect-idp': ['redirect-idp'],
+
+  'cancel-transaction': ['cancel-transaction'],
+};
+
+const getClassNameMapping = (appState) => {
+  const formName = appState.get('currentFormName');
+  let authenticatorType = appState.get('authenticatorType');
+  const recoveryAuthenticator = appState.get('recoveryAuthenticator');
+  let isPasswordRecoveryFlow = recoveryAuthenticator && recoveryAuthenticator.type === 'password';
+
+  if (authenticatorType === 'phone') {
+    // both sms and call have same type
+    authenticatorType = appState.get('currentAuthenticatorEnrollment') &&
```

Comment:
is this methodType? would prefer more explicitly like

```
if authenticator type is phone then
  key += `-${methodType}`
else if authenticatorType exists
  key += `-${authenticatorType}`

Reply:
yeah, authenticatorType is phone for sms and call so I had to look into the method[0].type